src/js/components/Institution.jsx

```diff
@@ -14,9 +14,44 @@ const Institution = ({
   submissions,
   onDownloadClick
 }) => {
-  if (!filing)
-    return (
-      <div className="usa-grid-full">
+  const status = submission && submission.status
+
+  return (
+    <div className="usa-grid-full">
+      {/*
+        a filing should be created when an institution is created
+        so this shouldn't happen but just in case ...
+        render the current status if there is a filing
+        otherwise render an alert
+      */}
+      {filing ? (
+        <section className="institution">
```

Comment:
Can we nest the check inside the section so we don't need to duplicate it?

Reply:
I wanted to, but [this](https://github.com/cfpb/hmda-platform-ui/pull/876/files/08c76507d1cc20a8ccfb90e000d9222b81f9791d#diff-7b62ba75715e7d849fd47779257044d6R47) being outside of the `current-status` is preventing it.